docs/README.md

```diff
@@ -1,11 +1,19 @@
 # Kueue documentation
 
+Welcome to Kueue - Kubernetes native APIs and controller for queueing jobs.
+
 ## Understand Kueue
 
 Learn about Kueue and its fundamental concepts.
 
 [**View concepts**](concepts)
 
+## Setup
+
+For a step-by-step instructions on how to get Kueue running on your cluster.
+
+[**View Setup**](setup)
+
 ## Learn how to use Kueue
 
 Look up common tasks and how to perform them using a short sequence
```

Comment:
```suggestion
Find common Kueue configuration tasks and learn how to perform them.
```

Reply:
It's not just configuration. Batch admins do configuration, but Batch users run and monitor their jobs.

Reply:
Gotcha. Maybe:
```
Learn what are and how to perform the available Kueue tasks, based on the user type.
```

Reply:
this was not part of the pr :)